src/utils/mediaQuery/index.d.ts

```diff
@@ -16,4 +17,6 @@ export type Devices =
   | "smallMobile";
 
 declare const MediaQuery: { [key in Devices]: QueryFunction };
-export { MediaQuery, MediaQuery as default };
+declare const GetBreakpointWidth: (name: string, theme: ThemeShape, pure?: boolean) => string;
```

Comment:
Should not be it called `getBreakpointWidth` so it corresponds with the actual name? 🤔